pyradiosky/utils.py

```diff
@@ -278,3 +280,113 @@ def download_gleam(path=".", filename="gleam.vot", overwrite=False, row_limit=No
     table.write(opath, format="votable", overwrite=overwrite)
 
     print("GLEAM catalog downloaded and saved to " + opath)
+
+
+def modified_gleam(
+    gleam_filename="gleamegc.dat",
+    usecols=(10, 12, 77, -5),
+    fill_blank=False,
+    nside=32,
+    add_peeled_sources=False,
+    modified_gleam_filename="",
+):
+    """
+    Write modified gleam catalogue to fill the blank regions, and also to add the peeled sources.
+
```

Comment:
Can you fill out this docstring a little more to describe the algorithm in a bit more detail? You can even put in a reference to your paper if you like. But other people are much more likely to use it if they understand what it's doing and what problem it's trying to solve.